operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakDeploymentTest.java

```diff
@@ -614,6 +614,61 @@ public void testUpgradeRecreatesPods() {
         }
     }
 
+    @Test
+    public void testHostnameDebugOption() {
```

Comment:
I think we don't need an Operator test for this feature. It's completely transparent to the Operator.